src/log.cpp

```diff
@@ -48,6 +48,7 @@ static uint64_t init_logging_groups() {
         {"api", loggroup::api},
         {"event", loggroup::event},
         {"validation", loggroup::validation},
+        {"cfg", loggroup::cfg},
```

Comment:
```suggestion
        {"config", loggroup::config},
```

Reply:
+1

Reply:
so there is another variable in this context called config hence why I used cfg  change it.

Reply:
Ah...
Here is a proposition: let's put the 2 enums (`loggroup` & `loglevel`) in a namespace `clvk_log` (I wanted to call it just `log` but it is already defined by the logarithmic function).
And because of the namespace let's call `loggroup` `group` and `loglevel` `level`?
Here is a patch of what I did that work on my machine.
[clvk-664.patch](https://github.com/kpet/clvk/files/14598498/clvk-664.patch)

Reply:
Ok, let's just keep cfg for simplicity then.